test/e2e/scrapeconfig_test.go

```diff
@@ -491,559 +491,578 @@ func testScrapeConfigDNSSDConfig(t *testing.T) {
-				KubernetesSDConfigs: []monitoringv1alpha1.KubernetesSDConfig{
-					{
-						Role: "Pod",
-						Selectors: []monitoringv1alpha1.K8SSelectorConfig{
-							{
-								Role:  "Pod",
-								Field: ptr.To(""),
-							},
+		expectedError: false,
+	},
+	{
+		name: "Selector Field with empty value",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+			KubernetesSDConfigs: []monitoringv1alpha1.KubernetesSDConfig{
+				{
+					Role: "Pod",
+					Selectors: []monitoringv1alpha1.K8SSelectorConfig{
+						{
+							Role:  "Pod",
+							Field: ptr.To(""),
 						},
 					},
 				},
 			},
-			expectedError: true,
 		},
-		{
-			name: "KubernetesSDConfigs: Selector Field with valid value",
-			scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
-				KubernetesSDConfigs: []monitoringv1alpha1.KubernetesSDConfig{
-					{
-						Role: "Pod",
-						Selectors: []monitoringv1alpha1.K8SSelectorConfig{
-							{
-								Role:  "Pod",
-								Field: ptr.To("metadata.name=foobar"),
-							},
+		expectedError: true,
+	},
+	{
+		name: "Selector Field with valid value",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+			KubernetesSDConfigs: []monitoringv1alpha1.KubernetesSDConfig{
+				{
+					Role: "Pod",
+					Selectors: []monitoringv1alpha1.K8SSelectorConfig{
+						{
+							Role:  "Pod",
+							Field: ptr.To("metadata.name=foobar"),
 						},
 					},
 				},
 			},
-			expectedError: false,
 		},
-		{
-			name: "KubernetesSDConfigs: Selector Field with valid value",
-			scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
-				KubernetesSDConfigs: []monitoringv1alpha1.KubernetesSDConfig{
-					{
-						Role: "Pod",
-						Selectors: []monitoringv1alpha1.K8SSelectorConfig{
-							{
-								Role:  "Pod",
-								Field: ptr.To("metadata.name=foobar"),
-							},
+		expectedError: false,
+	},
+	{
+		name: "Selector Field with valid value",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+			KubernetesSDConfigs: []monitoringv1alpha1.KubernetesSDConfig{
+				{
+					Role: "Pod",
+					Selectors: []monitoringv1alpha1.K8SSelectorConfig{
+						{
+							Role:  "Pod",
+							Field: ptr.To("metadata.name=foobar"),
 						},
 					},
 				},
 			},
-			expectedError: false,
 		},
-		{
-			name: "KubernetesSDConfigs: Selector Label and Field with duplicate values",
-			scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
-				KubernetesSDConfigs: []monitoringv1alpha1.KubernetesSDConfig{
-					{
-						Role: "Pod",
-						Selectors: []monitoringv1alpha1.K8SSelectorConfig{
-							{
-								Role:  "Pod",
-								Label: ptr.To("node.kubernetes.io/instance-type=master"),
-								Field: ptr.To("metadata.name=foobar"),
-							},
-							{
-								Role:  "Pod",
-								Label: ptr.To("node.kubernetes.io/instance-type=master"),
-								Field: ptr.To("metadata.name=foobar"),
-							},
+		expectedError: false,
+	},
+	{
+		name: "Selector Label and Field with duplicate values",
```

Comment:
nit: IMO Its good to prefix names with SD we are testing so its easy to identify, applies to all test names

Reply:
The sub-test should already be prefixed by the SD name.